simulation.py

```diff
@@ -0,0 +1,367 @@
+snowfall_zone = ""
+snowfall_zone_list = []
+
+for i in snowfall.index:
+    if (pd.isna(snowfall[i])):
+        snowfall_zone = "gray"
+    elif (snowfall[i] == 71) or (snowfall[i] == 73) or (snowfall[i] == 75) or (snowfall[i] == 77) or (snowfall[i] == 85) or (snowfall[i] == 86):
+        snowfall_zone = "red"
+    else:
+        snowfall_zone = "green"
+    snowfall_zone_list.append(snowfall_zone)
+    
+# Create a new DataFrame
+snowfall_zone_df_all = pd.DataFrame({
+    "Forecast Date": forecast_date,
+    "Forecast Time": forecast_time_et,
+    "Snowfall": snowfall,
+    "Snowfall Zone": snowfall_zone_list
+})
+
+print(snowfall_zone_df_all)
+
+    
+# freezing rain and freezing drizzle (both under freezing rain)
+freezing_rain_zone = ""
+freezing_rain = forecast_df["Weathercode"] 
+freezing_rain_zone_list = []
+
+for i in freezing_rain.index:
+    if (pd.isna(freezing_rain[i])):
+        freezing_rain_zone = "gray"
+    elif (freezing_rain[i] == 66) or (freezing_rain[i] == 67) or (freezing_rain[i] == 56) or (freezing_rain[i] == 57):
+        freezing_rain_zone = "red"
+    else:
+        freezing_rain_zone = "green"
+    freezing_rain_zone_list.append(freezing_rain_zone)
+
+
+# Create a new DataFrame
+freezing_rain_df_all = pd.DataFrame({
+    "Forecast Date": forecast_date,
+    "Forecast Time": forecast_time_et,
+    "Freezing Rain": freezing_rain,
+    "Freezing Rain Zone": freezing_rain_zone_list
+})
+
+print(freezing_rain_df_all)
+
+#gust (MAKE SURE TO DOWNLOAD CSV WITH UNITS AS KNOTS)
+wind_gusts_zone = ""
+wind_gusts = forecast_df["Wind Gusts"] 
+wind_gusts_zone_list = []
+
+for i in wind_gusts.index:
+    if (pd.isna(wind_gusts[i])):
+        wind_gusts_zone = "gray"    
+    elif (wind_gusts[i]) >= 30:
+        wind_gusts_zone = "red"
+    elif (wind_gusts[i] >= 25):
+        wind_gusts_zone = "yellow"
+    else:
+        wind_gusts_zone = "green"
+    wind_gusts_zone_list.append(wind_gusts_zone)
+
+
+# Create a new DataFrame
+wind_gusts_df_all = pd.DataFrame({
+    "Forecast Date": forecast_date,
+    "Forecast Time": forecast_time_et,
+    "Wind Gusts": wind_gusts,
+    "Wind Gusts Zone": wind_gusts_zone_list
+})
+
+print(wind_gusts_df_all)
+
+
+#temperature 
+temperature_zone = ""
+temperature = forecast_df["Temperature (°C)"] 
+temperature_zone_list = []
+for i in temperature.index:
+    if (pd.isna(temperature[i])):
+        temperature_zone = "gray"
+    elif (temperature[i] > 35) or (temperature[i] < -20):
+        temperature_zone = "red"
+    elif (temperature[i] > 30) or (temperature[i] < -10):
+        temperature_zone = "yellow"
+    else:
+        temperature_zone = "green"
+    temperature_zone_list.append(temperature_zone)
+# Create a new DataFrame
+temperature_df_all = pd.DataFrame({
+    "Forecast Date": forecast_date,
+    "Forecast Time": forecast_time_et,
+    "Temperature": temperature,
+    "Temperature Zone": temperature_zone_list
+})
+
+print(temperature_df_all)
+
+
+# lightning_potential_index_zone = ""
```

Comment:
What is going on with the lightning? Can we remove this if this will not be considered?